lib/daru/view/adapters/googlecharts/data_table_iruby.rb

```diff
@@ -62,7 +62,9 @@ def google_table_version
     end
 
     def package_name
-      'table'
+      return 'table' unless
```

Comment:
ChartEditor is going through `data_table_iruby.rb` :O . But why ?

Reply:
ChartEditor works for both google charts and tables, and code to generate the table is written in `data_table_iruby.rb`. `charteditor` package is loaded in Google Datatables when `chart_class` is set to Charteditor.

Reply:
Is there any example or documentation written, which describes this usecase ?

Reply:
I have written the rails examples in demo_daru-view and also in the method documentation [here](https://github.com/SciRuby/daru-view/pull/96/files#diff-cd0c826056999c96e0390d69150cc07bR129).

Reply:
Write examples in web apps and iruby as well, if haven't.